### PR TITLE
bottomless: migrate to auto-generated bindings

### PR DESCRIPTION
**Pull request by psarna**
This series migrates `bottomless` to the same auto-generated bindings that `sqld` already uses. Some bugs were fixed along the way. This series also comes with a smoke test for bottomless added to github actions.

### CODE REVIEW

**Review by MarinPostma — APPROVED** [review]
bors merge

**Comment by MarinPostma** [comment]
bors merge

**Comment by psarna** [comment]
sigh, no space left on device again

**Comment by psarna** [comment]
bors merge

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -87,6 +87,8 @@ jobs:
     runs-on: ubuntu-latest
     name: Run Tests
     steps:
+    - name: Make some room available
+      run: "sudo rm -rf /usr/local/lib/android && sudo rm -rf /usr/share/dotnet"
```
Comment (MarinPostma) [inline .github/workflows/rust.yml:91]: wth?

**Comment by MarinPostma** [comment]
bors merge

**Pull request merged**